substrate-node/pallets/pallet-tfgrid/src/tests.rs

```diff
@@ -1308,6 +1310,28 @@ fn test_create_and_update_policy() {
     });
 }
 
+#[test]
+ fn test_set_zos_version() {
```

Comment:
Might be good to add a test that sets a wrong version.

Reply:
oops, saw it after merging...
indeed, it would be better to test invalid version setting
follow case on new complement PR #409